analysis/line-of-sight-location/README.md

```diff
@@ -1,28 +1,32 @@
-# Line of Sight Location
+# Line of sight location
 
-Perform line of sight analysis in real-time.
+Perform a line of sight analysis between two points in real time.
+
+![Image of line of sight location](LineOfSightLocation.png)
```

Comment:
the original readme linked to the gif: just pointing this out, might be worth keeping it as the gif?

Reply:
Good catch. Happy to keep the gif, I think selecting the `png` is just a habit by now